### ISSUE TITLE
Getting a "Failed to decipher auth data." error, when trying to change password

### ISSUE THREAD

**Issue by afuuam** [mshanemc/sfdx-msm-plugins#3]
with reason as "Unsupported state or unable to authenticate data."

**Comment by mshanemc** [comment]
Are you still seeing this?  I've got hundreds of automated runs with passwords being set ok.  Maybe your org expired?

**Comment by afuuam** [comment]
Yes, I'm afraid. Creating a new scratch org works fine, but when trying to change the password returns the error.

**Comment by mshanemc** [comment]
Try cloning this repo and running the ./orgInit.sh script.  It works fine for me.  May be something interesting about your hub?

https://github.com/mshanemc/df17IntegrationWorkshops
